### ISSUE TITLE
 ui-payment > build > error

### ISSUE THREAD

**Issue by naddika** [oxygenpay/oxygen#35]
```
cd ui-payment
yarn install

[1/4] Resolving packages...
............................

$ husky install
husky - .git can't be found (see https://typicode.github.io/husky/#/?id=custom-directory)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

**Comment by swift1337** [comment]
Hey! Could you describe the use-case for building custom frontend?

**Comment by swift1337** [comment]
> Does it matter?

Yep, we want to improve developer experience by collecting use-cases outside of default setup (i.e. running via published docker image).

Regarding your question, you can follow gh action script to reproduce the same build:
- Put desired vars in `.env`
- Run `npm ci --ignore-scripts`
- Run `npx tsc && npx vite build --base='/p/'`

**Comment by swift1337** [comment]
```shell
 ▸ ~/Code/o2pay/oxygen/ui-dashboard ▸ git:develop ▸ nvm use                                                    
Found '/Users/abc/Code/o2pay/oxygen/ui-dashboard/.nvmrc' with version <16.13.0>
Now using node v16.13.0 (npm v8.1.0)
 ▸ ~/Code/o2pay/oxygen/ui-dashboard ▸ git:develop ▸ npm ci --ignore-scripts 

added 548 packages, and audited 549 packages in 31s

137 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
npm notice 
npm notice New major version of npm available! 8.1.0 -> 10.1.0
npm notice Changelog: https://github.com/npm/cli/releases/tag/v10.1.0
npm notice Run npm install -g npm@10.1.0 to update!
npm notice 
 ▸ ~/Code/o2pay/oxygen/ui-dashboard ▸ git:develop ▸ npx tsc && npx vite build --base='/p/' 
vite v4.4.7 building for production...
✓ 6482 modules transformed.
dist/assets/tron-5b6b6483.svg            0.43 kB │ gzip:   0.31 kB
dist/assets/eth-1f94df85.svg             0.53 kB │ gzip:   0.28 kB
dist/assets/arrow-up-d1a531a7.svg        0.76 kB │ gzip:   0.39 kB
dist/assets/arrow-down-ea43cf9f.svg      0.77 kB │ gzip:   0.39 kB
dist/assets/busd-a6ab749c.svg            0.86 kB │ gzip:   0.47 kB
dist/index.html                          0.90 kB │ gzip:   0.43 kB
dist/assets/matic-479d03ee.svg           1.02 kB │ gzip:   0.48 kB
dist/assets/usdt-337d36c2.svg            1.07 kB │ gzip:   0.52 kB
dist/assets/bnb-d068cbf5.svg             1.61 kB │ gzip:   0.73 kB
dist/assets/usdc-913adf09.svg            1.70 kB │ gzip:   0.70 kB
dist/assets/index-7de3f4c4.css           4.89 kB │ gzip:   1.41 kB
dist/assets/tron-167e02ed.js             0.62 kB │ gzip:   0.43 kB
dist/assets/eth-59de992b.js              0.81 kB │ gzip:   0.40 kB
dist/assets/arrow-up-49a1cfc5.js         0.92 kB │ gzip:   0.52 kB
dist/assets/arrow-down-ba5fc5fb.js       0.94 kB │ gzip:   0.51 kB
dist/assets/busd-88859388.js             1.02 kB │ gzip:   0.55 kB
dist/assets/matic-7f5c3fd9.js            1.21 kB │ gzip:   0.60 kB
dist/assets/usdt-a3321d3c.js             1.31 kB │ gzip:   0.64 kB
dist/assets/bnb-cb17ca65.js              1.69 kB │ gzip:   0.74 kB
dist/assets/usdc-6c2da4e2.js             1.89 kB │ gzip:   0.83 kB
dist/assets/index-78303a53.js        1,565.97 kB │ gzip: 489.28 kB

(!) Some chunks are larger than 500 kBs after minification. Consider:
- Using dynamic import() to code-split the application
- Use build.rollupOptions.output.manualChunks to improve chunking: https://rollupjs.org/configuration-options/#output-manualchunks
- Adjust chunk size limit for this warning via build.chunkSizeWarningLimit.
✓ built in 7.61s
 ▸ ~/Code/o2pay/oxygen/ui-dashboard ▸ git:develop ▸ 
```

**Comment by swift1337** [comment]
@naddika I want to warn that you attitude is a bit disrespectful.
